pkgs/servers/nextcloud/packages/30.json

```diff
@@ -139,6 +139,16 @@
       "agpl"
     ]
   },
+  "maps": {
+    "hash": "sha256-Rsg+26VNEX4lAHESfWQgbxvsAO/krvrml8pqBGfDXts=",
+    "url": "https://github.com/nextcloud/maps/releases/download/v1.5.0-1-nightly/maps-1.5.0-1-nightly.tar.gz",
```

Comment:
Argh right, I remember there was an issue here...
Will add to my todolist (expect high latency though).

In the meantime, what do the other maintainers thinK? Ship it or manually exclude it?